deploy/csi-operator.yaml

```diff
@@ -13,45 +13,6 @@
 ###########  CSI Node and Driver CRDs   ############
 ###########                             ############
 ####################################################
-
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Is this CRD no longer required?

Reply:
No  , this resource is deprecated in k8s against the CSINode resource.